packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -5433,19 +5421,24 @@ class InheritedElement extends ProxyElement {
 /// ### Specializing the getters
 ///
 /// [RenderObjectElement] objects spend much of their time acting as
-/// intermediaries between their [widget] and their [renderObject]. To make this
-/// more tractable, most [RenderObjectElement] subclasses override these getters
-/// so that they return the specific type that the element expects, e.g.:
+/// intermediaries between their [widget] and their [renderObject]. It is
```

Comment:
Can you also add a note to the docs of `Element.widget` explaining that it is totally expected to cast that at the callsite of the getter to the specific widget type of the element?

Reply:
done, PTAL